src/Stripe.net/Entities/PaymentMethods/PaymentMethod.cs

```diff
@@ -56,24 +89,64 @@ public Customer Customer
         internal ExpandableField<Customer> InternalCustomer { get; set; }
         #endregion
 
+        /// <summary>
+        /// Properties specific to Eps PaymentMethod.
```

Comment:
non blocking nit, but it looks like we call this EPS in the docs rather than Eps?

Reply:
We'll fix all of these with autogen anyways.

Reply:
yeah I think it doesn't really matter though I tried to use the right writing everywhere but I typed that string a lot by hand :p